<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="27" started-at="2015-12-09T20:06:22Z" finished-at="2015-12-09T20:06:22Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="27" started-at="2015-12-09T20:06:22Z" finished-at="2015-12-09T20:06:22Z">
      <class name="test1.Solution">
        <test-method status="FAIL" signature="running()[pri:0, instance:test1.Solution@5e025e70]" name="running" duration-ms="13" started-at="2015-12-09T14:06:22Z" finished-at="2015-12-09T14:06:22Z">
          <exception class="java.lang.IllegalAccessException">
            <message>
              <![CDATA[Class org.testng.internal.MethodInvocationHelper can not access a member of class test1.Solution with modifiers "public static"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalAccessException: Class org.testng.internal.MethodInvocationHelper can not access a member of class test1.Solution with modifiers "public static"
	at sun.reflect.Reflection.ensureMemberAccess(Unknown Source)
	at java.lang.reflect.AccessibleObject.slowCheckMemberAccess(Unknown Source)
	at java.lang.reflect.AccessibleObject.checkAccess(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalAccessException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- running -->
      </class> <!-- test1.Solution -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
